src/command_modules/azure-cli-interactive/tests/test_feedback.py

```diff
@@ -53,10 +53,10 @@ def test_heuristic(self):
     def test_update_freq(self):
         """ tests updating the files for frequency """
         fh.update_frequency = self.norm_update
+        fh.FREQUENCY_PATH = os.path.join(shell_config(), 'mock_freq.json')
```

Comment:
Use `mkstemp` for a truly temp file.
https://docs.python.org/2/library/tempfile.html#tempfile.mkstemp